src/assets/stylesheets/index.scss

```diff
@@ -356,3 +356,45 @@ body {
   }
 }
 
+#mc_embed_signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+#mc_embed_signup_scroll {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.subscribeBar) {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.subscribe-text) {
+  font-size: 14pt;
```

Comment:
nit: this makes the sign up text the biggest text on the page of its style. It breaks the visual hierarchy a bit, unless it is our intention to direct the user's eye to the sign up form, I would omit the font-size here and let it use the default page font size.